crates/ruff_python_parser/src/lexer.rs

```diff
@@ -169,7 +170,7 @@ impl<'source> Lexer<'source> {
     }
 
     /// Lex an identifier. Also used for keywords and string/bytes literals with a prefix.
-    fn lex_identifier(&mut self, first: char) -> Result<Tok, LexicalError> {
+    fn lex_identifier(&mut self, first: char, ascii_first_char: bool) -> Result<Tok, LexicalError> {
```

Comment:
nit: maybe `is_first_char_ascii` / `is_first_ascii`?

Reply:
I went for `first_char_is_ascii` in https://github.com/astral-sh/ruff/pull/10412/commits/d785be50606df110d848606165e2f7a9aa406835